lib/takeaway.rb

```diff
@@ -0,0 +1,19 @@
+# Allows takeaway to checkout current order and send confirmation text message using Twilio API
+require_relative 'send_sms' #twilio-ruby code hidden in .gitignore 
```

Comment:
Great that you're thinking about not exposing your api keys and other info by adding sensitive data to .gitignore.

Rather than ignoring the whole implementation of the twilio api, however, I'd suggest just setting up some variables containing the specific information you want to keep secure in a seperate file, ignoring that file and pulling the variables into you Message class code.